eos-install-firefox

```diff
@@ -0,0 +1,354 @@
+#!/bin/bash -e
+
+# Downloads Firefox and unzips to the user's home directory,
+# with a desktop launcher
+
+exit_with_error() {
```

Comment:
It would be a good idea to make sure that ${TEMP_DIR} gets removed whenever this function is called.